src/analyzers/index.js

```diff
@@ -0,0 +1,18 @@
+class Analyzer {
+  constructor(project) {
+    this._project = project;
+    this._domParser = new DOMParser();
+    const htmlString = this._project.getIn(['sources', 'html']);
+    this._doc = this._domParser.parseFromString(htmlString, 'text/html');
+  }
+
+  get containsExternalScript() {
+    return this._doc.querySelector('script');
```

Comment:
OK sorry to be such an obsessive freak here, but I think this should be a boolean… mind giving it the `Boolean()` treatment?